### PR TITLE
Add troubleshooting issue for 692

### PR DESCRIPTION
**Pull request by micgibso**
Signed-off-by: micgibso <micgibso@uk.ibm.com>

### CODE REVIEW

**Comment by micgibso** [comment]
@jagraj @GeekArthur @rajivnathan Review please.

File: docs/_documentations/troubleshooting.md [hunk 1]
```diff
@@ -363,6 +364,49 @@ These steps reproduce the issue:
...
+Che version: 7.2.0
+IDE extension version: 0.5.0
+IDE version: **7.1.0
+Action/Topic: ODO with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/692
+-->
+## ODO projects are not deleted after the workspace is deleted
+
+If you create ODO projects and then delete the workspace, all your ODO deployments still exist.
```
[the remark this answers is not in the document]
Comment (micgibso) [reply docs/_documentations/troubleshooting.md]: Clarified

File: docs/_documentations/troubleshooting.md [hunk 2]
```diff
@@ -363,6 +364,49 @@ These steps reproduce the issue:
...
+IDE version: **7.1.0
+Action/Topic: ODO with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/692
+-->
+## ODO projects are not deleted after the workspace is deleted
+
+If you create ODO projects and then delete the workspace without deleting the projects, all your ODO deployments still exist. 
+
+Follow these steps to reproduce the issue:
```
Comment (rajivnathan) [inline docs/_documentations/troubleshooting.md]: Thinking we don't need the `steps to reproduce the issue` section?

Comment (micgibso) [reply docs/_documentations/troubleshooting.md]: Have reduced to `These steps to reproduce the issue:` as a lead in sentence, which is more common amongst all the issues documented in this file.

**Pull request merged**